### ISSUE TITLE
Add disclosures for 2017 templates

### ISSUE THREAD

**Issue by willdwyer** [willdwyer/bcbsmaissuestracker#365]
Hello,

We have a new disclosure that will need to be stitched (mapped) with the new HMO Blue Select Plans (all segments).  Can you please add this disclosure to the templates for these plans?  The plan designs are noted below.

```
900390HR_HMO_Blue_Select_Ded_1000_80-0281SG1-1-17
900391HR_HMO_Blue_Select_Ded_1000_Copay_80-0282SG1-1-17
900392HR_HMO_Blue_Select_Ded_2000_80-0283SG1-1-17
900393HR_HMO_Blue_Select_Ded_2000_Copay_80-0284SG1-1-17
900394HR_HMO_Blue_Select_Ded_3000_80-0285SG1-1-17
900396HR_HMO_Blue_Select_Ded_1000_80-0281MM1-1-17
900397HR_HMO_Blue_Select_Ded_1000_Copay_80-0282MM1-1-17
900398HR_HMO_Blue_Select_Ded_2000_80-0283MM1-1-17
900399HR_HMO_Blue_Select_Ded_2000_Copay_80-0284MM1-1-17
900400HR_HMO_Blue_Select_Ded_3000_80-0285MM1-1-17
```

Thanks,
[HMO_Blue_Select_Disclosure.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/317309/HMO_Blue_Select_Disclosure.pdf)

**Comment by willdwyer** [comment]
Please use this disclosure (attached).

Also, please add disclosure to these templates as well (below)

900389HR_Access_Blue_Select_Saver_2000_80-0280SG1-1-17
900395HR_Access_Blue_Select_Saver_2000_80-0280MM1-1-17
900401HR_HMO_Blue_Select_Ded_1000_80-0286LG1-1-17
900402HR_Network_Blue_Select_Ded_80-0286LGASC1-1-17

[HMO_Blue_Select_Disclosure[1].pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/320614/HMO_Blue_Select_Disclosure.1.pdf)

**Comment by TimHeasley** [comment]
BCBSMA team will review and identify attributes to be used and added if necessary.

**Comment by willdwyer** [comment]
Hello,

For all the 2017 Select Blue plans (noted above), these have been assigned the attribute of: Product Identifier - Select Limited Network.  

We are proposing the following requirements:

*When a plan has attributes of:
1.Product Group Type - HMO Blue Local -or- Access Blue Local -or- Network Blue - or- HMO Blue NE -or- Access Blue NE -or- Network Blue NE
2. Product Identifier - Select Limited Network
*The plan will generate with the attached HMO Blue Select disclosure.
[HMO_Blue_Select_Disclosure[1].pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/323621/HMO_Blue_Select_Disclosure.1.pdf)

**Comment by TimHeasley** [comment]
Updated disclosure rules, tested & confirmed in PClient5, (900403 & 900399).
Ready for retest in PClient5.

**Comment by TimHeasley** [comment]
55-0645_Chapter 61_HMO_2015.pdf
55-0249 limited regional tiered provider_Consumer Guide.pdf

Hi Tim,

Please add the following 2 disclosures to the SG 1-50 HMO Blue Select plans (these 5 below). Please make the change in PCLIENT5, and we will proceed with testing.

900389HR_Access_Blue_Select_Saver_2000_80-0280SG1-1-17
900390HR_HMO_Blue_Select_Ded_1000_80-0281SG1-1-17
900391HR_HMO_Blue_Select_Ded_1000_Copay_80-0282SG1-1-17
900392HR_HMO_Blue_Select_Ded_2000_80-0283SG1-1-17
900393HR_HMO_Blue_Select_Ded_2000_Copay_80-0284SG1-1-17
900394HR_HMO_Blue_Select_Ded_3000_80-0285SG1-1-17

55-0645_Chapter 61_HMO_2015.pdf
55-0249 limited regional tiered provider_Consumer Guide.pdf

Disclosure rules have been updated, tested in PClient5 (800390) and appears correctly.
Ready for retest.

**Comment by willdwyer** [comment]
Looks good.  Please move to production.

**Comment by TimHeasley** [comment]
Disclosure rules & HMO_Blue_Select_Disclosure.pdf have been moved to Production. Other pdfs were already located in Production.

**Comment by willdwyer** [comment]
Hi Tim,

[55-0645_Chapter 61_HMO_2015.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/350461/55-0645_Chapter.61_HMO_2015.pdf)
[55-0646_Chapter 61_PPO_2015.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/350462/55-0646_Chapter.61_PPO_2015.pdf)
Please attach the following disclosures for the below referenced plan templates:

**55-0645**
900007HR_Access_Blue_NE_Basic_2000_80-0117HCCSSG1-1-17
900025HR_HMO_Blue_NE_Ded_500_80-0208HCCSSG1-1-17
900027HR_HMO_Blue_NE_Ded_1000_80-0114HCCSSG1-1-17
900030HR_HMO_Blue_NE_Ded_2000_80-0209HCCSSG1-1-17
900032HR_HMO_Blue_NE_Ded_3000_80-0240HCCSSG1-1-17
900034HR_HMO_Blue_NE_Ded_1000_Coins_80-0263HCCSSG1-1-17
900040HR_HMO_Blue_NE_Options_Ded_v.5_80-0220SG1-1-17
900041HR_HMO_Blue_NE_Options_Ded_II_v.5_80-0221SG1-1-17
900042HR_HMO_Blue_NE_Options_Ded_III_v.5_80-0258SG1-1-17
900044HR_HMO_Blue_NE_Premier_Value_Coins_80-0112SG1-1-17
900045HR_HMO_Blue_NE_Premier_Value_Coins_80-0112HCCSSG1-1-17
900046HR_HMO_Blue_NE_Premier_Value_80-0111HCCSSG1-1-17
900371HR_HMO_Blue_NE_Ded_1500_80-0270HCCSSG1-1-17
900372HR_HMO_Blue_NE_Ded_1000_Copay_80-0272HCCSSG1-1-17
900389HR_Access_Blue_Select_Saver_2000_80-0280SG1-1-17
900390HR_HMO_Blue_Select_Ded_1000_80-0281SG1-1-17
900391HR_HMO_Blue_Select_Ded_1000_Copay_80-0282SG1-1-17
900392HR_HMO_Blue_Select_Ded_2000_80-0283SG1-1-17
900393HR_HMO_Blue_Select_Ded_2000_Copay_80-0284SG1-1-17
900394HR_HMO_Blue_Select_Ded_3000_80-0285SG1-1-17

**55-0646**
900049HR_Preferred_Blue_Ded_500_80-0259HCCSSG1-1-17
900051HR_Preferred_Blue_Ded_1000_80-0211HCCSSG1-1-17
900053HR_Preferred_Blue_Ded_2000_80-0119HCCSSG1-1-17
900055HR_Preferred_Blue_Basic_2000_80-0122HCCSSG1-1-17
900059HR_Preferred_Blue_Options_v.5_80-0222SG1-1-17
900373HR_Preferred_Blue_Options_Ded II_v.5_80-0278SG1-1-17
900375HR_Preferred_Blue_Options_Ded III_v.5_80-0279SG1-1-17

**Comment by TimHeasley** [comment]
Will,
The rules use attributes to map the disclosures to the correct plans.  They are not mapped individually.

For the 55-0645_Chapter 61_HMO_2015 disclosure the current mapping is:
Market Segment - Small Group 1-50, 
Provider Network Tier - 3 or 4 tier
Product Type - HMO
The following plans from above do not have the correct attributes:
900007HR_Access_Blue_NE_Basic_2000_80-0117HCCSSG1-1-17
900044HR_HMO_Blue_NE_Premier_Value_Coins_80-0112SG1-1-17
900389HR_Access_Blue_Select_Saver_2000_80-0280SG1-1-17
900390HR_HMO_Blue_Select_Ded_1000_80-0281SG1-1-17
900391HR_HMO_Blue_Select_Ded_1000_Copay_80-0282SG1-1-17
900392HR_HMO_Blue_Select_Ded_2000_80-0283SG1-1-17
900393HR_HMO_Blue_Select_Ded_2000_Copay_80-0284SG1-1-17
900394HR_HMO_Blue_Select_Ded_3000_80-0285SG1-1-17

For the 55-0646_Chapter 61_PPO_2015 disclosure the current mapping is:
Market Segment - Small Group 1-50, 
Provider Network Tier - 3 or 4 tier
Product Type - PPO
All of the PPO plans above have the correct attributes and appear in Search.

I did verify that the pdf's you attached are current in Production.
Let me know if you have any questions.
Thanks,
Tim

**Comment by willdwyer** [comment]
Hi Tim,

Per last comment, please disregard 900044HR_HMO_Blue_NE_Premier_Value_Coins_80-0112SG1-1-17 from the list.  That plan should not have that disclosure attached to it.

Also, per disclosure mapping... for the 55-0645 disclosure, regard Product Type - HMO _and_ HMO-Type as the same, for disclosure purposes.

Also, is it possible to change the disclosure mapping for this to rely on Product Identifier instead of Product Network Tier?  I.e. if a plan has a Product Identifier attribute of HCCS then it would prompt the disclosure to appear, instead of Product Network Tier of 3.

**Comment by TimHeasley** [comment]
Will,
Yes, we can change the disclosure mapping to rely on Product Identifier.
I'll update the mapping to include Product Type - HMO and HMO-Type
I can also change the mapping to use Product Identifier attribute of HCCS and remove the Product Network Tier of 3. Do you want any changes to the Tier 4 mapping?
Let me know and I can proceed with the updates in PClient5 so you can test before moving to Prod.
Thanks, Tim

**Comment by willdwyer** [comment]
Tim,

Please change the mapping to use Product Identifier attributes HCCS, Tiered Network and Select Limited Network.

Thanks,

**Comment by TimHeasley** [comment]
Will,
PPO is tested and working in PClient5.  I confirmed both HCCS & Tiered Product Identifiers have the correct disclosure appearing.  Ready for retest in PClient5.
I am continuing work on the HMO plans and plan on having them ready for you this afternoon.
Thanks, Tim

**Comment by TimHeasley** [comment]
Will,
All Product Types are ready for retest in PClient5 - HMO, HMO-Type, and PPO.
Thanks, Tim

**Comment by willdwyer** [comment]
Hi Tim,

The SG 1-50 Select Blue plans are missing the Limited Regional Consumer Guide disclosure (55-0249), please include these with them.  The job numbers to these plan designs are listed below.

900389
900390
900391
900392
900393
900394

**Comment by TimHeasley** [comment]
The current mapping for **55-0249_limited_regional_tiered_provider_Consumer_Guide** is
Market Segment-Small Group 1-50
Provider Network Tier-3 or 4

Do you want the same mapping as 55-0645?

**Comment by willdwyer** [comment]
Yes, please.

**Comment by willdwyer** [comment]
(and 55-0645 as well).

If the 55-0645 disclosure appears, then the 55-0249 disclosure should always appear as well.

**Comment by TimHeasley** [comment]
Ready for retest in PClient5.
The mapping is the same for both 55-0645 & 55-0249:
Market Segment-Small Group
Product Identifier-HCCS, Tiered Network and Select Limited Network.
Product Type-HMO or HMO Type

**Comment by willdwyer** [comment]
Please move this to production.

**Comment by michparrish** [comment]
Moved to Prod. Please let me know if all looks ok.
Thank you

**Comment by willdwyer** [comment]
Hi Michelle,

All looks good, but we will have 1 additional disclosure that will be added to ALL 2017 templates.  More to come on this.  For now this will be held in Monitoring status.

**Comment by michparrish** [comment]
I will hold off on updating the Disclosure mapping in Prod then, for all 2017 disclosures?

**Comment by michparrish** [comment]
I feel like this is overlapping with the other 2017 Disclosure request item. is it?

**Comment by willdwyer** [comment]
#365 was intended for adding additional disclosures (not replace prior versions), while #369 was intended for replacing newer versions of disclosure templates over prior versions.  We can certainly merge these into 1 item, though.  (I will do that now and re-word #369)

**Comment by michparrish** [comment]
ok thank you!

**Comment by willdwyer** [comment]
Merged into #369
